scripts/parallel_deblur.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+from os import listdir
+from os.path import dirname, join, isfile, exists
+from glob import glob
+
+from skbio.parse.sequences import parse_fasta
+from biom.util import biom_open, HAVE_H5PY
+from qiime.util import split_sequence_file_on_sample_ids_to_files
```

Comment:
if writing a new method is only to avoid import from qiime, we'll also need a solution for ```
from qiime.parallel.util import ParallelWrapper``` and ```from qiime.util import write_biom_table```